docs/platform/release/2-4-6.md

```diff
@@ -0,0 +1,35 @@
+---
+id: 2-4-6
```

Comment:
could we also add dates for the release pages?

Reply:
I added them to the title, I hope that is what you meant.

Reply:
@ivan-deity I'd prefer to have it more subtle (not as a part of the page title, so maybe as a smaller subtitle), but I'd imagine that would require more work for the templates - so it's fine

Reply:
@Borales, it's not a problem, I will try to add it today, no worries :))